di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -264,4 +267,12 @@ private String getUserIdValue(String userIdTextValue) {
 
         return URN_UUID + UUID.randomUUID();
     }
+
+    private String getSignInJourneyId(String signInJourneyId) {
+        if (signInJourneyId.isEmpty()) {
+            return URN_UUID + UUID.randomUUID();
+        }
+
+        return signInJourneyId;
+    }
```

Comment:
Wasn't sure if this was necessary (if other services can use `/authorize` indirectly)